build.sbt

```diff
@@ -86,6 +86,22 @@ ThisBuild / githubWorkflowBuildPreamble := Seq(
   ),
 )
 
+ThisBuild / githubWorkflowBuild := {
+  (ThisBuild / githubWorkflowBuild).value ++ WorkflowJob(
+    "testSbtPlugin",
+    "Test sbt plugin",
+    List(
+      WorkflowStep.Use(UseRef.Public("coursier", "setup-action", "v1")),
+      WorkflowStep.Run(
+        commands = List(s"sbt zioHttpGenSbt/scripted"),
+        name = Some("Test sbt plugin"),
+        cond = Some("${{ github.event_name == 'pull_request' }}"),
+      ),
+    ),
+    scalas = List(Scala212),
+  ).steps
+}
```

Comment:
wasteful.
I wanted to add sbt testing to CI, and this works. but…
it has scalaVersion set to 2.12 (only supporting sbt 1.x for now),
and it runs for every matrix value. which means we run the same tests multiple times.
Not sure how to opt out.